src/nyc_trees/apps/core/tasks.py

```diff
@@ -70,3 +72,11 @@ def _class_for_name(module_name, class_name):
 def _model_from_fq_name(fq_name):
     module_name, class_name = fq_name.rsplit('.', 1)
     return _class_for_name(module_name, class_name)
+
+
```

Comment:
FYI you can omit `bind=True` and the `self` argument if you don't need to use `self`.